tests/foreman/ui/test_dashboard.py

```diff
@@ -14,13 +14,57 @@
 
 @Upstream: No
 """
+from robottelo.constants import ANY_CONTEXT
 from robottelo.decorators import stubbed, tier1, tier2
 from robottelo.test import UITestCase
+from robottelo.ui.factory import set_context
+from robottelo.ui.locators import common_locators
+from robottelo.ui.session import Session
 
 
 class DashboardTestCase(UITestCase):
     """Tests for Dashboard UI"""
 
+    @tier1
+    def test_positive_search_random(self):
+        """Perform search on Dashboard using any random string
+
+        @id: 28062a97-d642-41ac-b107-0b8a41eac478
+
+        @Steps:
+
+        1. Navigate to Monitor -> Dashboard
+        2. Perform search using any random test string
+
+        @BZ: 1391365
+
+        @Assert: Check that we have zero as a result of search and any error is
+        not raised
+        """
+        with Session(self.browser) as session:
+            set_context(session, org=ANY_CONTEXT['org'])
+            self.assertEqual(self.dashboard.search('test_string'), '0')
```

Comment:
It's a nitpick of course, but `test_string` isn't a random string :) How about using one of our valid names generator altogether with subtest? That way the test would use a random string, each time of different valid type, and basically could check all the possible string types in case we decide to roll back `run_one_datapoint` property?